content/OTP/OTPs_Explained.adoc

```diff
@@ -24,21 +24,21 @@ will allow the validation server to know which OTPs have already been used.
 
 image:otp_details.png[]
 
-The outputed OTP of the YubiKey OTP is provided in the Modhex characterset. The Modhex characterset uses characters common across the majority of latin alphabet QWERTY keyboard layouts, allowing for functionality regardless of the language set.
+The YubiKey OTP is encoded in modhex (Modified Hexadecimal), a format that mostly allows for functionality regardless of the language set. Modhex uses only characters whose position on the keyboard does not change across the majority of latin-script keyboard layouts that are similar to QWERTY.
```

Comment:
This feels a bit awkward to me. How about something like this?

```suggestion
The YubiKey OTP is encoded in modhex (Modified Hexadecimal), a format designed for maximum compatibility across host language settings. Modhex uses characters whose position on the keyboard is the same between many latin-script keyboard layouts similar to QWERTY.
```

Reply:
Cool, that looks like an improvement to me.